static/src/stylesheets/module/atoms/_youtube.scss

```diff
@@ -158,4 +164,74 @@
             bottom: 8px;
         }
     }
+
+    .immersive-main-media__media & {
+        // & = .youtube-media-atom
+        padding-bottom: 0;
+        position: absolute;
+        top: 0;
+        left: 0;
+        right: 0;
+        bottom: 0;
+        height: 100%;
+        z-index: 0;
+        &__immersive-interface {
```

Comment:
Now you've commented '// & = .youtube-media-atom'  this selector will compile to .immersive-main-media__media__immersive-interface...that class doesn't exist, from what I can see you want that to be youtube-media-atom__immersive-interface. Same goes for following selectors '__play-button', '_bottom-bar__duration' etc.